lib/cookie.js

```diff
@@ -1070,6 +1117,24 @@ CookieJar.prototype.setCookie = function(cookie, url, options, cb) {
     return cb(options.ignoreError ? null : err);
   }
 
+  /* 6265bis-02 S5.4 Steps 15 & 16 */
+  /*  TODO: unsure whether this ignore error condition should check for options.ignoreError
+      || or && with this.prefixSecurity?
+   */
+  const ignoreErrorForPrefixSecurity = this.prefixSecurity === PrefixSecurityEnum.SILENT;
```

Comment:
Any suggestions on how to confirm what the right behavior should be here? I'll try and review the RFC on this point, unless you have a better idea. :)

Reply:
Perhaps we could augment the unit test around this as well; I'd like to see a case for PrefixSecurityEnum.SILENT.